module/VuFind/src/VuFind/View/Helper/Root/MakeLink.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Make link view helper
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2020.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+namespace VuFind\View\Helper\Root;
+
+use Laminas\View\Helper\AbstractHelper;
+
+/**
+ * Make link view helper
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class MakeLink extends AbstractHelper
+{
+    /**
+     * Create an anchor tag (or just text if no href provided)
+     *
+     * @param string $text  Link contents
```

Comment:
This phpDoc doesn't reflect some of the ways parameters are actually allowed to be passed to this function; it's probably worth documenting all the legal uses.

Reply:
That includes $href accepting string|array (and using arrays as attributes ... maybe the param should actually be renamed $hrefOrAttrs) and $attrs accepting string|array (and treating strings as 'class' by default).